### PR TITLE
Improve documentation of Control.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

**Comment by groud** [comment]
The content of the changes are good in general. But I am not sure about the wording. 
Maybe someone from the documentation team could have a look ? @NathanLovato ?

File: doc/classes/Control.xml [hunk 1]
```diff
@@ -921,16 +932,19 @@
 			The control will receive mouse button input events through [method _gui_input] if clicked on. These events are automatically marked as handled and they will not propagate further to other controls.
 		</constant>
 		<constant name="MOUSE_FILTER_PASS" value="1" enum="MouseFilter">
-			The control will receive mouse button input events through [method _gui_input] if clicked on. If this control does not handle the event, the parent control (if any) will be considered for a mouse click, and so on until there is no more parent control to potentially handle it. Even if no control handled it at all, the event will still be handled automatically.
+			The control will receive mouse button input events through [method _gui_input] if clicked on. If this control does not handle the event, the parent control (if any) will be considered for a mouse click, and so on until there is no more parent control to potentially handle it. Even if no control handled it at all, the event will still be handled automatically (unhandled input will not fire).
```
Comment (NathanLovato) [inline doc/classes/Control.xml]: Remove the parentheses.

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Thanks for your work! This needed some update. There´s almost never a need for parentheses in docs writing. They create nested sentences and make the content harder to read. Use a short sentence or a proposition instead.

**Comment by akien-mga** [comment]
> This needed some update. There´s almost never a need for parentheses in docs writing. They create nested sentences and make the content harder to read. Use a short sentence or a proposition instead.

Bump :)

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**